test/models/inbox_notice_test.rb

```diff
@@ -12,19 +12,22 @@ def notification
   end
 
   test 'requires a level' do
-    skip "One of: :info, :warn, :success, :danger"
+    skip 'One of: :info, :warn, :success, :danger'
   end
 
   test 'requires a state' do
-    skip "One of: :pending, :sent, :delivered, :unread, :read"
+    skip 'One of: :pending, :sent, :delivered, :unread, :read'
   end
 
   test 'notifications' do
-    skip "Notification event log, like NotificationEvent :sms, :number, :sent_at"
+    skip """
+      Notification event log, like NotificationEvent
+      :sms, :number, :sent_at
+    """
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.